python/grass/jupyter/timeseries.py

```diff
@@ -0,0 +1,274 @@
+        if not test:
+            raise NameError(
+                _(
+                    "Could not find space time raster or vector " "dataset named {}"
+                ).format(timeseries)
+            )
+
+        # Create a temporary directory for our PNG images
+        self._tmpdir = (
+            # pylint: disable=consider-using-with
+            tempfile.TemporaryDirectory()
+        )
+
+        def cleanup(tmpdir):
+            tmpdir.cleanup()
+
+        weakref.finalize(self, cleanup, self._tmpdir)
+
+        # create list of layers to render and date/times
+        self._renderlist, self._dates = collect_lyr_dates(self.timeseries, self._etype)
+        self._date_name_dict = {
+            self._dates[i]: self._renderlist[i] for i in range(len(self._dates))
+        }
+
+    def d_legend(self, **kwargs):
+        """Wrapper for d.legend. Passes keyword arguments to d.legend in render_layers
+        ethod.
+        """
+        self._legend = True
+        self._legend_kwargs = kwargs
+
+    def render_layers(self):
+        """Renders map for each time-step in space-time dataset and save to PNG
+        image in temporary directory.
+
+        Must be run before creating a visualization (i.e. time_slider or animate).
+
+        Can be time-consuming to run with large space-time datasets.
+        """
+
+        for name in self._renderlist:
+            # Create image file
+            filename = os.path.join(self._tmpdir.name, f"{name}.png")
+            # self._filenames.append(filename)
+
+            # Render image
+            img = GrassRenderer(filename=filename)
+            if self.basemap:
+                img.d_rast(map=self.basemap)
+            if self._etype == "strds":
+                img.d_rast(map=name)
+            elif self._etype == "stvds":
+                img.d_vect(map=name)
+            if self.overlay:
+                img.d_vect(map=self.overlay)
+            # Add legend if called
+            if self._legend:
+                info = gs.parse_command("t.info", input="precip_sum", flags="g")
+                min_min = info["min_min"]
+                max_max = info["max_max"]
+                img.d_legend(
+                    raster=name, range=f"{min_min}, {max_max}", **self._legend_kwargs
+                )
+
+    def time_slider(self, slider_width="60%"):
+        """
+        Create interactive timeline slider.
+
+        param str slider_width: width of datetime selection slider as a
+                                percentage (%) or pixels (px)
+        """
+        # Lazy Imports
+        import ipywidgets as widgets  # pylint: disable=import-outside-toplevel
+        from IPython.display import Image  # pylint: disable=import-outside-toplevel
+
+        # Datetime selection slider
+        slider = widgets.SelectionSlider(
+            options=self._dates,
+            value=self._dates[0],
+            description="Date/Time",
```

Comment:
Should be translatable.

Reply:
`description=_("Date/Time")` should solve this, right?